src/app/app.component.html

```diff
@@ -1,12 +1,12 @@
-<div>
+<div *ngIf="isDataAvailable">
 
   <app-header></app-header>
 
   <section class="app-content">
 
     <!-- Message Strip section -->
     <section>
-      <app-message></app-message>
+      <app-message gate="{{boardingGate}}"></app-message>
```

Comment:
If app-message has an input then you should use `[gate]="boardingGate"` instead
```suggestion
      <app-message [gate]="boardingGate"></app-message>
```